hrms/payroll/doctype/income_tax_slab/income_tax_slab.js

```diff
@@ -1,8 +1,23 @@
 // Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and contributors
 // For license information, please see license.txt
 
-frappe.ui.form.on('Income Tax Slab', {
-	currency: function(frm) {
+frappe.ui.form.on("Income Tax Slab", {
+	refresh: function (frm) {
+		if (frm.doc.__islocal) return;
```

Comment:
Only show this button if the docstatus = 1

Reply:
Also except for the create button in the Salary Component, we can make all the "Create" buttons primary stressing on the "next" action.